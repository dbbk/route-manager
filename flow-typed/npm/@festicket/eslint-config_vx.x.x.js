// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
