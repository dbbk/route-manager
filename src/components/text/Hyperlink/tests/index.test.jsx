// @flow
import React from 'react';
import { mount } from 'enzyme';
import { Hyperlink } from '@festicket/react-ui-components';

describe('<Hyperlink />', () => {
  test('with initial props', () => {
    const tree = mount(<Hyperlink size="regular">Regular size test</Hyperlink>);
    expect(tree).toMatchSnapshot();
  });

  test('with element prop', () => {
    const tree = mount(<Hyperlink element="span">Small size test</Hyperlink>);
    expect(tree).toMatchSnapshot();
  });
});
