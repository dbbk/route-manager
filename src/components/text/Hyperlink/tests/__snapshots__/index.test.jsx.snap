// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hyperlink /> with initial props 1`] = `
.c0 {
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
  text-decoration: underline;
  font-size: 18px;
  line-height: 1.12;
}

<Hyperlink
  as="p"
  color="black"
  size="regular"
>
  <TextPrimitive
    color="black"
    element="p"
    size="regular"
    spacing="md"
    underline={true}
    variant="p"
    weight="regular"
  >
    <Styled(Primitive)
      color="black"
      element="p"
      size="regular"
      spacing="md"
      underline={true}
      weight="regular"
    >
      <Primitive
        className="c0"
        color="black"
        element="p"
        size="regular"
        spacing="md"
        underline={true}
        weight="regular"
      >
        <p
          className="c0"
        >
          Regular size
        </p>
      </Primitive>
    </Styled(Primitive)>
  </TextPrimitive>
</Hyperlink>
`;

exports[`<Hyperlink /> with size prop 1`] = `
.c0 {
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
  text-decoration: underline;
  font-size: 16px;
  line-height: 1.12;
}

<Hyperlink
  as="p"
  color="black"
  size="small"
>
  <TextPrimitive
    color="black"
    element="p"
    size="small"
    spacing="md"
    underline={true}
    variant="p"
    weight="regular"
  >
    <Styled(Primitive)
      color="black"
      element="p"
      size="small"
      spacing="md"
      underline={true}
      weight="regular"
    >
      <Primitive
        className="c0"
        color="black"
        element="p"
        size="small"
        spacing="md"
        underline={true}
        weight="regular"
      >
        <p
          className="c0"
        >
          Small size
        </p>
      </Primitive>
    </Styled(Primitive)>
  </TextPrimitive>
</Hyperlink>
`;
